src/main/java/seedu/address/logic/parser/volunteercommandparser/VolunteerNewCommandParser.java

```diff
@@ -60,7 +60,7 @@ public VolunteerAddCommand parse(String args) throws ParseException {
         Time endTime = VolunteerParserUtil.parseTime(argMultimap.getValue(VOLUNTEER_PREFIX_END_TIME).get());
```

Comment:
Might want to ensure that start time is before end time here too, similar to Event

Reply:
as time is removed, this validation is no longer needed for volunteer. thanks for pointing out the issue tho!